apps/common/src/main/scala/org/lfdecentralizedtrust/splice/setup/ParticipantPartyMigrator.scala

```diff
@@ -342,7 +343,7 @@ class ParticipantPartyMigrator(
   ): Future[Unit] = {
     for {
       _ <- participantAdminConnection.disconnectFromAllDomains()
-      _ <- Future.traverse(partyIds) { partyId =>
+      _ <- MonadUtil.sequentialTraverse(partyIds) { partyId =>
```

Comment:
Can we slap a big do not change to parallel on top of it?

Reply:
done